src/MapKit/MKEnums.cs

```diff
@@ -168,6 +175,58 @@ public enum MKLocalSearchResultType : ulong
 		PointOfInterest = 1 << 1,
 	}
 
+	[Mac (13,0), iOS (16,0), MacCatalyst (16,0), NoWatch, TV (16,0)]
+	[Native]
+	public enum MKDirectionsRoutePreference : long
+	{
+		Any = 0,
+		Avoid
```

Comment:
nit, add a comma to reduce the diff in the future.
```suggestion
		Avoid,
```